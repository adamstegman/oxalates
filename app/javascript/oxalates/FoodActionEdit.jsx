import React from 'react';
import PropTypes from 'prop-types';

export const FoodActionEdit = ({ onEditFoods }) => {
  return (
    <button type="button"
            className="food-action"
            onClick={onEditFoods}>
      Edit
    </button>
  );
};

FoodActionEdit.propTypes = {
  onEditFoods: PropTypes.func.isRequired,
};
