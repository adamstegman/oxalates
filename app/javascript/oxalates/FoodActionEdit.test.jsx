import React from 'react';
import renderer from 'react-test-renderer';

import { FoodActionEdit } from './FoodActionEdit';

test('FoodActionEdit renders the edit foods button', () => {
  const component = renderer.create(
    <FoodActionEdit onEditFoods={() => {}} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
