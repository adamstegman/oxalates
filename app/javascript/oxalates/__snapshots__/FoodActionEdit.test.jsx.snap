// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodActionEdit renders the edit foods button 1`] = `
<button
  className="food-action"
  onClick={[Function]}
  type="button"
>
  Edit
</button>
`;
